### ISSUE TITLE
Spike: Define best practice for external IdP's

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#21]
### Expected result

A best practice of using external identity providers in MedioClinic.

### Additional context

LearningKit/docs already contain a howto, however, it was made as straightforward as possible, without any specifics project-wise.

### Resources

* https://docs.kentico.com/13/developing-websites/developing-xperience-applications-using-asp-net-core/integrating-asp-net-core-identity/adding-external-authentication-using-asp-net-core-identity
* https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/?view=aspnetcore-3.1&tabs=visual-studio

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
